docs/reference/ilm/actions/ilm-allocate.asciidoc

```diff
@@ -0,0 +1,118 @@
+[[ilm-allocate]]
+=== Allocate
+
+Phases allowed: warm, cold.
+
+The Allocate action allows you to specify which nodes are allowed to host the
+shards of the index and set the number of replicas.
+Behind the scenes, it is modifying the index settings
+for shard filtering and/or replica counts. When updating the number of replicas,
```

Comment:
Maybe link to these settings if possible?

Reply:
The following "more information" links reference the most relevant info.